params/version.go

```diff
@@ -33,12 +33,14 @@ var Version = func() string {
 }()
 
 // VersionWithMeta holds the textual version string including the metadata.
-var VersionWithMeta = func() string {
+var VersionWithMeta, Unstable = func() (string, bool) {
```

Comment:
I don't think you need to separate out this extra variable `Unstable`. You can just compare `params.Version` with `"unstable"` at the call site.